apps/web/components/apps/OmniInstallAppButton.tsx

```diff
@@ -18,10 +18,11 @@ export default function OmniInstallAppButton({ appId, className }: { appId: stri
   const utils = trpc.useContext();
 
   const mutation = useAddAppMutation(null, {
-    onSuccess: () => {
+    onSuccess: (data: any) => {
       //TODO: viewer.appById might be replaced with viewer.apps so that a single query needs to be invalidated.
       utils.viewer.appById.invalidate({ appId });
       utils.viewer.apps.invalidate({ extendsFeature: "EventType" });
+      if (data.setupPending) return;
```

Comment:
If there's setup pending we dont show the message about successfully installing the app